### PR TITLE
Update cors

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: config/cors.php [hunk 1]
```diff
@@ -10,37 +10,20 @@
...
-        'allowedOrigins'      => ['*'],
-        'allowedHeaders'      => [],
-        'allowedMethods'      => [],
-        'exposedHeaders'      => [],
-        'maxAge'              => 0,
-        'hosts'               => [],
-    ],
-
-    'paths' => [
-        'api/v1/*' => [
```
Comment (GrahamCampbell) [inline config/cors.php]: This will surely cause us issues?

Comment (jbrooksuk) [inline config/cors.php]: I added the Cors middleware to the `api` middleware group. I don't know if that fixes it 100%?

**Comment by GrahamCampbell** [comment]
> Merge branch 'master' into update-cors

Oops.

**Comment by GrahamCampbell** [comment]
Rebased and merged.

**Comment by jbrooksuk** [comment]
:+1:

**Pull request merged**